scripts/nemo/canary/test_1b_flash.py

```
@@ -0,0 +1,303 @@
+            else:
+                t = " "
+                idx = int(fields[0])
+
+            id2token[idx] = t
+            token2id[t] = idx
+
+    model = OnnxModel(args.encoder, args.decoder)
+
+    fbank = create_fbank()
+
+    start = time.time()
```

Flake8 (7.3.0)

[error] local variable 'start' is assigned to but never used

(F841)